<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternSelection</class>
 <widget class="QWidget" name="PatternSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>203</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pattern Selection</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QListView" name="color_list_lv">
       <property name="isWrapping" stdset="0">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="clear_list_pb">
       <property name="text">
        <string>Clear List</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="clear_selected_pb">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Clear Selected</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="done_pb">
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="cancel_pb">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
